mnd/mnd/forms/widgets/mnd_widgets.py

```diff
@@ -0,0 +1,62 @@
+from django.urls import reverse_lazy
```

Comment:
Similar bug as with multi-cdefiles where if you navigate away from a saved form it asks for confirmation